### ISSUE TITLE
[data request] Cifar 100 corrupted data set

### ISSUE THREAD

**Issue by iswariyam** [tensorflow/datasets#2151]
* Name of dataset: Cifar 100 corrupted
* URL of dataset: https://github.com/hendrycks/robustness
* License of dataset: Apache License 2.0
* Short description of dataset and use case(s): The dataset contains corrupted versions of the Cifar 100 test set and is based on the paper [Benchmarking Neural Network Robustness to Common Corruptions and Perturbations](https://arxiv.org/abs/1903.12261) Dan Hendrycks and Thomas Dietterich. 19 different corruptions are applied to the 10000 images in Cifar 100 test set and these corruptions are available in 5 different intensities. This is a benchmark dataset used for evaluating robustness of image classifiers.

Folks who would also like to see this dataset in `tensorflow/datasets`, please thumbs-up so the developers can know which requests to prioritize.

And if you'd like to contribute the dataset (thank you!), see our [guide to adding a dataset](https://github.com/tensorflow/datasets/blob/master/docs/add_dataset.md).

**Comment by iswariyam** [comment]
Tensorflow datasets already has the corrupted version of Cifar 10 in its catalog. It would be useful to have this version of Cifar 100 also.

**Comment by vedantsingh1404** [comment]
I would like to contribute to this!

**Comment by vijayphoenix** [comment]
@vedantsingh1404 A PR for this dataset already exists https://github.com/tensorflow/datasets/pull/2152.
Maybe you could try adding some other dataset

**Comment by vedantsingh1404** [comment]
Sure, I didn't find the dataset in the dataset list though.
